Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/eventregistration/organizer/details/QrCodeDetailFragment.kt

```diff
@@ -17,15 +19,24 @@ import de.rki.coronawarnapp.util.ui.observe2
 import de.rki.coronawarnapp.util.ui.popBackStack
 import de.rki.coronawarnapp.util.ui.viewBindingLazy
 import de.rki.coronawarnapp.util.viewmodel.CWAViewModelFactoryProvider
-import de.rki.coronawarnapp.util.viewmodel.cwaViewModels
+import de.rki.coronawarnapp.util.viewmodel.cwaViewModelsAssisted
 import javax.inject.Inject
 import kotlin.math.abs
 
 class QrCodeDetailFragment : Fragment(R.layout.trace_location_organizer_qr_code_detail_fragment), AutoInject {
 
     @Inject lateinit var viewModelFactory: CWAViewModelFactoryProvider.Factory
 
-    private val vm: QrCodeDetailViewModel by cwaViewModels { viewModelFactory }
+    private val navArgs by navArgs<QrCodeDetailFragmentArgs>()
+
+    private val vm: QrCodeDetailViewModel by cwaViewModelsAssisted(
```

Comment:
I think we are trying to streamline it not with "vm", but with "viewModel". Would be nice to rename it 🏷️